### DISCUSSION TITLE
project not working

### DISCUSSION THREAD

**Discussion by 14digital** [norkunas/youtube-dl-php#184] [General]
hi ,

 i have follow the steps to install youtube-dl plus composer
but facing the issue 

i have atthached the screenshot of code with xampp.

please help me how to resolve the issue.

thanks .

![Screenshot 2022-07-05 at 12 33 08 AM](https://user-images.githubusercontent.com/65076267/177209631-4965a492-cbc9-473d-9397-525732cae62c.png)

![Screenshot 2022-07-05 at 12 32 49 AM](https://user-images.githubusercontent.com/65076267/177209619-811d87b8-6498-4979-b244-7b229688fed8.png)

![Screenshot 2022-07-06 at 12 51 20 PM](https://user-images.githubusercontent.com/65076267/177498934-13a83399-6034-41ff-adac-3f1ae3c707d6.png)

**Comment by norkunas** [comment]
Hello, and what issue are you facing?

**Reply by 14digital** [reply]
Hi Norkunas, 

Whenever I load download.php/index.php in xampp it give an error... The error Image is added above.

Please guide me how to make this function.. I am new to php.

**Reply by norkunas** [reply]
You should check your php logs what's the error if you are not using any error handler in php

**Reply by norkunas** [reply]
but I think the problem will be that you haven't configured proper downloads path
